code-challenges.js

```diff
@@ -21,22 +21,109 @@ const temp1 = 42
+            return `${temp}F is above boiling point!`
+        } else if (temp === 212){
+            return `${temp}F is at boiling point!`
+        } else {
+            return `${temp}F is below boiling point!`
+        }
+    }
+            console.log(boilingPoint(temp1))
+            console.log(boilingPoint(temp2))
+            console.log(boilingPoint(temp3))
+
+                //Output ---> 42F is below boiling point
+                //Output ---> 350F is above boiling point
+                //Output ---> 212F is at boiling point 
+                //Also tried with several other numbers all were represented with expected outputs 
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+// for my assessment i will use built in methods for these two arrays instead of building a function 
+    //In order to see the two arrays in one we will assign a var named bothArrays to the following step 
+    //the built in method .concat() allows us to combine two arrays         
+        //Once they are merged we can use the other built in method .length to count the amount of indexs in the array
+        //We will console.log() bothArrays.length
+
+        var bothArrays = myNumbers1.concat(myNumbers2)
+        console.log(bothArrays.length)
+
+        //Output ---> 10 
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 eilrahC"
 
 const currentCohort = "Charlie 2022"
 
+    //Create a function with arrow syntax to cycle through any string and reverse the order of the letters
+        //Make a function named reverseString with one argument named string 
+            //This will only work if the user inputs a string
+        //We need to split the string into an array using .split(' ') in order to reverse it 
+            // "hi" --> ["h", "i"]
+            // I assigned the split string to a varriable called splitString
+        //We then need to reverse the order of the array using .reverse()
+            // ["h", "i"] ---> ["i", "h"]
+            //I assigned the reversed array to a new variable named reverseArray
+        //We then need to transform that array back to a string 
+            // ["i", "h"] ---> "ih"
+            // In order to do this I assigned the reversed array to a variable called joinedString 
+        //Finally return joinedString
+        //And console.log(reverseString(currentCohort))
+
+const reverseString = (string) => {
+
+    var splitString = string.split('')
+    var reverseArray = splitString.reverse()
+    var joinedString = reverseArray.join('')
+    return joinedString
+}
+
+console.log(reverseString(currentCohort))
+
+//      Output ---> "2202 eilrahC"
+//      I also tried an actual string instead of a variable and it also worked
```

Comment:
Good work! I like this style of declaring variables for each step / transformation.